libs/core/src/lib/button/button.component.ts

```diff
@@ -37,7 +37,8 @@ import { applyCssClass } from '@fundamental-ngx/core/utils';
     changeDetection: ChangeDetectionStrategy.OnPush,
     host: {
         '[attr.type]': 'type',
-        '[attr.disabled]': '_disabled || null'
+        '[attr.disabled]': '_disabled || null',
+        '[attr.aria-label]': 'ariaLabel ? ariaLabel : label !== undefined ? label +","+ fdType : glyph === undefined || glyph === null  ? title : glyph.split("-").join(" ") +","+ fdType ' 
```

Comment:
why do we need this?

Reply:
@mikerodonnell89 
https://github.com/SAP/fundamental-ngx/pull/6385#pullrequestreview-742950708
requested for this change.